drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/loaders/reason_for_exit_loader.rb

```diff
@@ -42,11 +42,26 @@ def build_records
           next # early return
         end
 
+        # reason for exit is required, but blank values are present in csv
+        reason_for_exit = row_value(row, field: 'ReasonForExit', required: false)
+        # voluntary_termination_value is required, but blank values are present in csv
+        voluntary_termination_yn = row_value(row, field: 'VoluntaryTermination', required: false)
+
+        # Actual voluntary/involuntary status. First try to infer from the reason, then look at the Y/N given.
+        voluntary_termination = if reason_for_exit.present?
+          voluntary_reason?(reason_for_exit)
+        elsif voluntary_termination_yn.present?
```

Comment:
note, present? isn't needed since voluntary_termination_yn normalizes empty string to nil.